services/brig/src/Brig/Team/DB.hs

```diff
@@ -162,5 +163,8 @@ countInvitations t = fromMaybe 0 . fmap runIdentity <$>
     cqlSelect :: PrepQuery R (Identity TeamId) (Identity Int64)
     cqlSelect = "SELECT count(*) FROM team_invitation WHERE team = ?"
 
+-- | FUTUREWORK: two weeks after https://github.com/wearezeta/backend-issues/issues/775 has been
+-- deployed, we won't find any `Nothing` role values any more; then it will be safe to change the
```

Comment:
> So Maybe Role / defaultRole should stay here.

I like this principle.

I don't like this principle applied inconsistently, however. Sometimes we leave stuff in, sometimes we deprecate things (like managed convs) but make it fairly easy for people to reenable them, sometimes we just don't communicate anything to anybody at all (like when a DB migration tool has to be run manually and we just write a tool but don't write anywhere that it has to be used).

If we decide to apply this principle consistently then I'd like everyone to explicitly agree on that because otherwise I have no idea what I'm allowed / not allowed to do and what I should / shouldn't look for when reviewing other people's PRs.